src/VisualStudioExtension/QsharpLibTemplate/Library.qs

```diff
@@ -1,9 +1,10 @@
-﻿namespace Quantum.$safeprojectname$
-{
-    open Microsoft.Quantum.Intrinsic;
+﻿namespace Quantum.$safeprojectname$ {
+
     open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Intrinsic;
     
+
     operation HelloQ () : Unit {
```

Comment:
Not related to this PR sorry, but do we want to take this as an opportunity to follow the style guide here?
```suggestion
    operation SayHello() : Unit {
```

Reply:
Pretty sure this would break sanity tests...